Lint-Whoppers/nutshell
# New Articles are Posting to the DOM

# Description

git fetch --all
git checkout cjNewArticleEventListener

Fixes # (issue)

Check to make sure that after you submit the new article form (when you click the new article button, scroll to the bottom of the page to see and fill out the form) you can see your entry on the DOM. 


Please delete options that are not relevant.

- [ x] New feature (non-breaking change which adds functionality)

# Testing Instructions

git fetch --all
git checkout cjNewArticleEventListener

Check to make sure that after you submit the new article form (when you click the new article button, scroll to the bottom of the page to see and fill out the form) you can see your entry on the DOM. 

# Checklist:

- [x ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [x ] My changes generate no new warnings or errors
- [ x] I have added test instructions that prove my fix is effective or that my feature works
